pkg/arvo/sys/vane/clay.hoon

```diff
@@ -1575,12 +1555,13 @@
         |-  ^+  ..park
         ?~  wux  ..park
         =/  rov  rove.wove.i.wux
-        =?    qyx
-            ?+  -.rov  |
-              %sing  ?=([%a * %app %publish %hoon ~] mood.rov)
-              %next  ?=([%a * %app %publish %hoon ~] mood.rov)
-            ==
-          (~(del by qyx) wove.i.wux)
+        ::  XX: revive
```

Comment:
OTOH, the upgrade's already happened and has to be done stepwise anyway, so we can probably just get rid of the whole +migrate-dist.